Library/Homebrew/sandbox.rb

```diff
@@ -5,12 +5,16 @@
 require "io/console"
 require "pty"
 require "tempfile"
+require "utils/fork"
 
 # Helper class for running a sub-process inside of a sandboxed environment.
 class Sandbox
   SANDBOX_EXEC = "/usr/bin/sandbox-exec"
   private_constant :SANDBOX_EXEC
 
+  TIOCSCTTY = 0x20007461
```

Comment:
Can be post-merge but: a comment and/or better variable name here would be good, I have no ideal what this means.